docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -716,152 +737,119 @@ Date: Thu, 23 Aug 2018 11:57:17 GMT
 
 [{"id":1,"name":"Secretlessly Fluffy"}]
 </pre>
-We expect the request to respond with a JSON array containing all the pets.
 
-There we have it. This application is communicating with a target service without managing any secrets.
+We should get a 200 response with a JSON array of the pets.
 
-## Rotate Target Service Credentials
+That's it! 
 
-In this section, you get to see how an application using the Secretless Broker deals with credential rotation.
-These are the steps you wil take to rotate the credentials of the dabatase:
-+ rotate application-user DB-credentials in the database
-+ update the application-user DB-credentials in the vault
-+ prune existing connections established using old credentials
+The application is connecting to a password-protected Postgres database **without
```

Comment:
is there some way to emphasize this point? would it be worth coming back to the original image and emphasizing the message we're trying to send with the tutorial using the image again?